src/main/cpp/feedback/DriverFeedback.cpp

```diff
@@ -100,16 +100,20 @@ void DriverFeedback::UpdateDiagnosticLEDs()
 {
 
     StateMgr *taleStateManager = MechanismConfigMgr::GetInstance()->GetCurrentConfig()->GetMechanism(MechanismTypes::DRAGON_TALE);
-    auto taleMgr = taleStateManager != nullptr ? dynamic_cast<DragonTale *>(taleStateManager) : nullptr;
     StateMgr *intakeStateManager = MechanismConfigMgr::GetInstance()->GetCurrentConfig()->GetMechanism(MechanismTypes::INTAKE_MANAGER);
-    auto intakeMgr = intakeStateManager != nullptr ? dynamic_cast<IntakeManager *>(intakeStateManager) : nullptr;
-    if (taleMgr != nullptr)
+    if (taleStateManager != nullptr && intakeStateManager != nullptr)
```

Comment:
Not this, you need to check if the mechanismConfigMgr ... ->GetCurrentConfig() is a null pointer and then remove the if statement. You already check the null pointer for each manager
 MechanismConfigMgr::GetInstance()->GetCurrentConfig() != nullptr